src/main/java/sx/blah/discord/api/internal/DiscordClientImpl.java

```diff
@@ -312,15 +317,15 @@ public void login() {
 		String gateway = obtainGateway();
 		new RequestBuilder(this).setAsync(true).doAction(() -> {
 			if (shard != null) {
-				ShardImpl shardObj = new ShardImpl(this, gateway, new int[]{shard[0], shard[1]});
+				ShardImpl shardObj = new ShardImpl(this, gateway, new int[]{shard[0], shard[1]}, identifyPresence);
```

Comment:
The presence set in the client builder can be defined as "the presence set when the client logs in." So if the client is logged in again, it will use that presence. I think it's a non-issue.